versioned_docs/version-2.11.x/admin-api-topics.md

```diff
@@ -522,6 +522,28 @@ admin.topics().peekMessages(topic, subName, numMessages);
 
 </TabItem>
 
+</Tabs>
```

Comment:
Seems you targeted 2.11.x file, but your PIP implementation is targeted at master which is 3.2, no?